src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/storage/LeaseMetaDataChecker.java

```diff
@@ -72,4 +73,14 @@ private boolean isMetaDataLeaseExpired(String leaseExpirationTime) {
                 .isBefore(LocalDateTime.now(EUROPE_LONDON_ZONE_ID)); // check if lease expired
         }
     }
+
+    public void clearMetaData(BlobClient blobClient, String leaseId) {
+        Map<String, String> blobMetaData = new HashMap<>();
+        blobMetaData.put(LEASE_EXPIRATION_TIME, null);
+        blobClient.setMetadataWithResponse(
+            blobMetaData,
```

Comment:
can we not just pass null here instead of map with expiry time null ?

Reply:
make sense. let's try.